src/components/LanguageSelector/LanguageSelector.tsx

```diff
@@ -14,6 +14,9 @@ export type LanguageSelectorProps = {
   label?: string
   langs: LanguageDefinition[]
   small?: boolean
+  /**
+   * Custom classes for the call-to-action
+   */
```

Comment:
ℹ️ This shows in Storybook: 

![CleanShot 2024-04-18 at 15 09 03@2x](https://github.com/trussworks/react-uswds/assets/371943/6d16da04-65dd-48e8-ac89-9872efd8b4fc)